hw/ip/rstmgr/data/rstmgr.hjson

```diff
@@ -31,7 +31,7 @@
       type:    "uni",
       name:    "resets",
       act:     "req",
-      width:   1,            // temporary work-around
+      width:   1,            // temporary work-around pending #2023
```

Comment:
@eunchan 
just a temp work-around, i didn't want to hack up the `intermodule.py` code since you've already fixed it.